_test/template.response.test.php

```diff
@@ -314,6 +321,17 @@ public function test_response_has_notify_message_when_five_hundred_pages_have_no
         $this->assertEquals($expectedNotifyMorePages, $notifyMorePages, 'Expected Notification message when pages are higher then the maximum amount of pages does not match.');
     }
 
+    public function test_plugin_is_sorted_at_the_top() {
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $expectedSortValue = 1;
+        $this->assertEquals($expectedSortValue, $adminPlugin->getMenuSort());
+    }
+
+    public function test_plugin_not_requires_to_be_superuser() {
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $this->assertTrue($adminPlugin->forAdminOnly());
```

Comment:
The test method says `not requires to be superuser`, yet it checks that it is for `adminOnly`. Is admin < superuser or is the test wrong?

Reply:
DokuWiki does not display this very good. I think the method name is kinda misleading. As I explained in the DocBlock for the method of the Plugin:

```php
/**
* * **true** => You do not need to be superuser to access this plugin.
* * **false** => You do need to be superuser to access this plugin.
*
* @return bool See method description.
*
*/
```

The method should be renamed to `accessForAdmins()`. As it is a DokuWiki function, renaming is not possible.

Reply:
Worst method name ever. 😒